src/Error.cpp

```diff
@@ -172,6 +201,11 @@ ErrorReport::~ErrorReport() noexcept(false) {
     }
 
 #ifdef HALIDE_WITH_EXCEPTIONS
+    static std::once_flag init;
+    std::call_once(init, []() {
+        std::set_terminate(UnhandledExceptionHandler);
```

Comment:
I don't think this belongs in libHalide... library code generally shouldn't trample consumers' termination handlers. Between these two locations, I think we'll have everything we care about covered.

1. Enabling this in GenGen
2. Enabling this in our tests via an object library, the same way we do the abort-to-exit-code-1 translation for our error tests in `expect_abort.cpp`

Reply:
Done, PTAL